test/TranslationLayer.UnitTests/ConsoleParametersTests.cs

```diff
@@ -26,5 +26,25 @@ public void LogFilePathShouldEnsureDoubleQuote()
 
             Assert.IsTrue(result.StartsWith("\""));
         }
+
+        [TestMethod]
+        public void OtherConsoleArgumentsShouldPersistValue()
```

Comment:
What if I give invalid arguments ?

Reply:
Invalid aArguments are not processed and understood here. that job is done by vstest.console.exe